gpii/node_modules/settingsHandlers/src/WebSocketsSettingsHandler.js

```diff
@@ -0,0 +1,154 @@
+/*!
+  GPII Web Sockets Settings Handler
+
+  Copyright 2014 Emergya
+
+  Licensed under the New BSD license. You may not use this file except in
+  compliance with this License.
+
+  You may obtain a copy of the License at
+  https://github.com/gpii/universal/LICENSE.txt
+*/
+
+/*global require*/
+
+(function () {
+
+    "use strict";
+
+    var fluid = require("infusion"),
+        gpii = fluid.registerNamespace("gpii");
+
+    fluid.registerNamespace("gpii.settingsHandlers");
+    fluid.registerNamespace("gpii.settingsHandlers.WebSocketsSettingsHandler");
+
+    // This global object keeps in memory the connected clients.
+    // Clients are added and removed respectively through addBrowser and
+    // removeBrowser functions
+    //
+    gpii.settingsHandlers.WebSocketsSettingsHandler.clients = {};
+
+    // This global object stores the current settings for every browser
+    //
+    gpii.settingsHandlers.WebSocketsSettingsHandler.settings = {};
+
+    gpii.settingsHandlers.WebSocketsSettingsHandler.getSettings = function (payload, id) {
+        var result = fluid.copy(payload);
+
+        for (var i=0; i< payload.length; i++) {
```

Comment:
Please could you lint with the settings in KASPARNET's https://github.com/GPII/universal/pull/290 so there are fewer conflicts when it goes in